crypto/BRCryptoWalletManager.c

```diff
@@ -555,13 +568,33 @@ cwmWalletEventAsETH (BREthereumClientContext context,
             if (NULL == wallet) {
                 BREthereumToken token = ewmWalletGetToken (ewm, wid);
```

Comment:
Are we leaking anything here (currency, unit)?

Reply:
I don't think so.  The `currency` and `unit` values are explicitly given.  The `wallet` value is taken enough times such that the last event callback has the wallet taken once.  (event handler is responsible to give all arguments)